go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Twingate/terraform-provider-twingate
 
-go 1.19
+go 1.20
```

Comment:
@ekampf I guess in github workflows we also would need to updated go version

Reply:
@vmanilo setup-go reads it from go.mod

Reply:
```
    - name: Set up Go
      uses: actions/setup-go@v3
      with:
        go-version-file: 'go.mod'
        cache: true
      id: go
```

Reply:
yeah right, I mean this line

https://github.com/Twingate/terraform-provider-twingate/blob/main/.github/workflows/release.yml#L31

Reply:
and readme https://github.com/Twingate/terraform-provider-twingate/blob/main/README.md?plain=1#L11